### DISCUSSION TITLE
Auto importing packages

### DISCUSSION THREAD

**Discussion by bq-wrongway** [helix-editor/helix#6007] [Q&A]
I dont know if this is possible in helix, but for example in vs code or sublime, when you use (lets say rust/go) and want to for example read the file : 

`let file = File::open(filename).unwrap(); `
i can on save, or manually trigger import of `use std::fs::File;`

is there such option in helix ?

**Comment by the-mikedavis** [comment]
This is not yet implemented: https://github.com/helix-editor/helix/issues/1565

**Reply by pascalkuthe** [reply]
you can manually trigger the import by hovering over file tough an pressing `<space>a` (and then selecting the option to import `std::fs::File`

**Reply by jumtp** [reply]
if auto import all miss packages . how to do ?

**Comment by bq-wrongway** [comment]
Thanks,now at least i know proper term for that!
